AppCUI/include/AppCUI.hpp

```diff
@@ -3299,8 +3299,11 @@ namespace Controls
     };
     enum class ViewerFlags : uint32
     {
-        None   = 0,
-        Border = 0x000100,
+        None          = 0,
+        HScroll       = 0x000010,
+        VScroll       = 0x000020,
+        Border        = 0x000100,
+        HideScrollBar = 0x000200
```

Comment:
Inteleg unde sunt necesare, dar nu inteleg logica. Daca nu pun HScroll si VScroll nu inseamna ca nu am scroll bar ? si in cazul asta, de ce as mai avea nevoie de HideScrollBar ?/
Mai vad si un risc la flagurile care duplica un flag de tipul GATTR_xxxx (daca le schimbam pe alea / ca si valoare) se schimba functionalitatea.

Reply:
yup, fixed